docs/overview/README.md

```diff
@@ -0,0 +1,58 @@
+<!--ONE-DOCS-HIDE start-->
+[👈 Return to README](../README.md)
+<!--ONE-DOCS-HIDE end-->
+
+# Welcome to One App
+
+As applications start to scale, and you start adding more developers to the project and new teams are created, it becomes increasingly difficult to release with confidence and to effectively collaborate without requiring large amounts of coordination and communication between teams. 
+
+One App helps solve this problem by providing a framework to build quality, modularized web applications. It helps you break the "Frontend Monolith" and split your application via independently developed, tested, and deployed **Micro Frontends** in the form of **Holocron Modules**.
+
+# Benefits
+
+The benefits to using One App include:
+
+- **Micro Frontends:** Powered by Holocron, One App allows for code splitting via independently developed, tested, and deployed Modules.
+- **Decoupled Codebases:** Each Holocron module has its own repository, reducing large and overwhelming codebases and helping developers to understand the code better. This also avoids accidental coupling by setting specific boundaries.
+- **Security:** Configurable security settings, all pages are protected by a content security policy and all modules are injected with sub-resource integrity.
+- **Flexibility:** An omnichannel solution to UI. Whether you're performing serverside, clientside, delivering modules as a service to web email or phone/IoT clients. One App covers this for you.
```

Comment:
>Whether you're performing serverside, clientside, delivering modules as a service to web email or phone/IoT clients.

Performing what server-side or client-side? Also, I believe they should be `server-side` and `client-side` instead of `serverside` and `clientside`

Reply:
comes from the home page https://one-amex-docs.americanexpress.com/ needs changing there as well

Reply:
updated